addons/mail/static/src/models/thread/thread.js

```diff
@@ -402,6 +440,13 @@ function factory(dependencies) {
             this.update({ areAttachmentsLoaded: true });
         }
 
+        /**
+         * Fetches info from this channel.
+         */
+        async fetchInfo() {
+            await this.env.models['mail.thread'].channelInfo({ ids: [this.id] });
+        }
```

Comment:
function makes no sense with non-channel thread, so would put a guard just in case.